spec/unit/knife/configure_spec.rb

```diff
@@ -157,9 +157,9 @@
     expect(::File).to receive(:open).with("/home/you/.chef/credentials", "w").and_yield config_file
     @knife.config[:repository] = "/home/you/chef-repo"
     @knife.run
-    expect(config_file.string).to match(/^client_name[\s]+=[\s]+'#{Etc.getlogin}'$/)
-    expect(config_file.string).to match(%r{^client_key[\s]+=[\s]+'/home/you/.chef/#{Etc.getlogin}.pem'$})
-    expect(config_file.string).to match(/^chef_server_url\s+=[\s]+'#{default_server_url}'$/)
+    expect(config_file.string).to match(/^client_name\s+=\s+'#{Etc.getlogin}'$/)
+    expect(config_file.string).to match(%r{^client_ky[\s=[\s+'/home/you/.chef/#{Etc.getlogin}.pem'$})
```

Comment:
```suggestion
    expect(config_file.string).to match(%r{^client_key\s+=\s+'/home/you/.chef/#{Etc.getlogin}.pem'$})
```

This one got pretty mangled as well.